python/tvm/relay/frontend/tensorflow.py

```diff
@@ -897,6 +897,7 @@ def _impl(inputs, attr, params):
                       disables=['momentum'])(inputs, attr)
 
         if need_cast:
+            out = _expr.TupleGetItem(out.astuple(), 0)
```

Comment:
Is this related to float16?

Reply:
I converted a float16 tf model containing the FusedBatchNorm operator. 
For the float16 model, the output of the FusedBatchNorm layer is converted back to float16(there may be other quantization types).
But the FusedBatchNorm layer has 3 outputs, if passed directly to the Cast layer will cause an exception error. 
So this modification only takes its first calculation output and passes it to the Cast

Reply:
It's true. I too see this recently and above change should fix it.